modules/stopping-microshift-service.adoc

```diff
@@ -0,0 +1,30 @@
+= Stopping the MicroShift service
+
+To stop the MicroShift service run:
+
+[source,terminal]
+----
+sudo systemctl stop microshift
+----
+
+Note that even though the MicroShift service has been stopped, workloads deployed on MicroShift will still keep running. You can confirm this with:
+
+[source,terminal]
+----
+sudo crictl ps -a
+----
+
+To also stop the deployed workloads, run:
+
+[source,terminal]
+----
+sudo systemctl stop kubepod.slice
+----
+
+To configure your RHEL machine to not start MicroShift automatically when the machine boots run:
+
+[source,terminal]
+----
+sudo systemctl disables microshift
+----
```

Comment:
Do we need to add a mention of the cleanup script? I think we might.

Reply:
IDK. I've always perceived the clean-up script as a MicroShift developer tool only (which is why it lives in `/hack` after all). It's also "nuke everything or nothing" and we'd probably need to rework and rename it if we wanted to make it a user-facing tool.

Reply:
We already rename it when RPM is installed. It's called `clean-all-microshift-data`

Reply:
Correct. And I'm arguing we should remove it or at least not document it.

Reply:
Yes, this script is good for support. So, it should not be documented here - rather in the Troubleshooting section, I think